### DISCUSSION TITLE
Error create the UI tab in SD

### DISCUSSION THREAD

**Discussion by kiingroup** [volotat/SD-CN-Animation#106] [Q&A] [answered]
is show the error when run the SD. anyone can help?

-------------------------------------------------------------------------------------------------------------------------------------------

Error executing callback ui_tabs_callback for D:\Git\stable-diffusion-webui\extensions\SD-CN-Animation\scripts\base_ui.py
Traceback (most recent call last):
  File "D:\Git\stable-diffusion-webui\modules\script_callbacks.py", line 119, in ui_tabs_callback
    res += c.callback() or []
  File "D:\Git\stable-diffusion-webui\extensions\SD-CN-Animation\scripts\base_ui.py", line 210, in on_ui_tabs
    export_setting_json = gr.Code(value='')
AttributeError: module 'gradio' has no attribute 'Code'

add tab
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
----------------------------------------------------------------------------------------------------------------------------------------------

**Comment by volotat** [comment, marked as answer]
Please update webui to the latest version. The most likely reason of the issue that you have older version of gradio.

**Reply by kiingroup** [reply]
thanks! fixed after update the webui.
